src/states.c

```diff
@@ -64,7 +64,7 @@ void logic()
 
 void draw()
 {
-	clearScreen();
+	//clearScreen();
```

Comment:
Don't leave commented out code like that.
It appears that clearScreen() has become redundant, as in both STATE_TITLE and STATE_GAME the whole screen area gets repainted with drawBackground() anyway, so we can safely remove this call from the code.
But how does it fix tearing on your platform?

Reply:
Bittboy does not have vsync and apparently the screen is being cleared directly after clearScreen() call. This is a probable reason of black horizontal bars showing in random positions on the screen. After removing that call, screen tearing is not visible to that extent and becomes acceptable (at least for me). Lack of vsync is a known issue on Bittboy.